excelParser.js

```diff
@@ -9,12 +9,10 @@ function extractFiles(path) {
 		deferred = defer();
 
 	var files = {
-		'xl/worksheets/sheet1.xml': {
-			deferred: defer()
-		},
 		'xl/sharedStrings.xml': {
 			deferred: defer()
-		}
+		},
```

Comment:
Need to keep track of file counts - used to parse through each discovered sheet.